recipes/swisslandstats-geopy/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
It looks like the license file is already in the source distribution so this copy can be removed.

Reply:
I have removed it from this repo in [f6cd132](https://github.com/conda-forge/staged-recipes/pull/12431/commits/f6cd132a13bb36e8794c1aa62b1c5bda79c9f9bf)